src/help.go

```diff
@@ -142,6 +142,7 @@ const (
 	DescOpen               = "open a file in the appropriate filemanager or default browser"
 	DescUrl                = "returns the url of each file"
 	DescVerbose            = "show step by step information verbosely"
+	DescFixClashes         = "fix clashes by renaming files. WARNING - this may be racy"
```

Comment:
Unfortunately I don't understand how Etags can be used here. Could you elaborate?